autogpt/prompt.py

```diff
@@ -38,6 +38,9 @@ def get_prompt() -> str:
     prompt_generator.add_constraint(
         'Exclusively use the commands listed in double quotes e.g. "command name"'
     )
+    prompt_generator.add_constraint(
+        "Use subprocesses for commands that will not terminate within a few minutes"
```

Comment:
I think the constraint wording is backwards

Reply:
It's correct. If the command would terminate within a few minutes (aka quickly) you can use the normal execute command. If it would take more than a few minutes, use subprocess.